### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Omriruvio**
Content and style for home page is done.
Fonts are included but do not currently match the design for some reason - could not stay to investigate, and this should probably be handled in the scope of the project rather than my specific branch.

Lets go team! ;)

### CODE REVIEW

**Comment by Omriruvio** [comment]
Fonts fixed, disregard previous comment.

File: home/pages/home.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import url('../vendor/normalize.css');
-@import url('../vendor/fonts.css');
+@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;500;700;900&family=Inter:wght@400;600;900&display=swap');
```
Comment (Sashikbear) [inline home/pages/home.css:2]: did exactly the same thing parallel to you and yet I still think that the 900 font doesn't work. Please checkout my PR as well

File: home/blocks/button/_type/button_type_view-all.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.button_type_view-all {
+  margin-top: 27px;
+  color: #00a8cc;
+  cursor: pointer;
+  font-size: 16px;
+  line-height: 23.5px;
```
Comment (Sashikbear) [inline home/blocks/button/_type/button_type_view-all.css:6]: shouldn't we use relative measurements instead of px for line height?

**Review by Sashikbear — APPROVED** [review]

File: home/blocks/button/_type/button_type_view-all.css (hunk 2 above)
Comment (Omriruvio) [reply home/blocks/button/_type/button_type_view-all.css:6]: we should

**Pull request merged**